app/views/layouts/tylium.html.erb

```diff
@@ -94,6 +94,10 @@
       <%= render partial: 'tags/modals/form' %>
     <% end %>
 
+    <% if defined?(Dradis::Pro) && (controller_name == 'issues' && action_name == 'show') %>
+      <%= render partial: 'dradis/pro/plugins/issuelib/issues/diff_modal' %>
+    <% end %>
+
```

Comment:
```suggestion
```

Let's leave this out. We have other modals we are only rendering in Pro by simply omitting them in the CE version of this file.